### PR TITLE
feat: Add guild member searching

### PR DESCRIPTION
**Pull request by ryanzhoudev**
Copies implementation from guild territory searching

### CODE REVIEW

File: src/main/java/com/wynntils/modules/utilities/configs/UtilitiesConfig.java [hunk 1]
```diff
@@ -86,6 +86,12 @@
     @Setting(displayName = "Searched Item Highlight Colour", description = "What colour should the highlight for searched items be?\n\n§aClick the coloured box to open the colour wheel.", order = 23)
     public CustomColor guildTerritoryMenuSearchHighlightColor = new CustomColor(80, 242, 242);
 
+    @Setting(displayName = "Show Guild Manage Member Search Bar", description = "Should the search bar be shown in the guild manage members GUI?\n\n§8Members that match the search will be highlighted.", order = 24)
+    public boolean showGuildMemberManageSearchbar = true;
+
+    @Setting(displayName = "Searched Item Highlight Colour", description = "What colour should the highlight for searched items be?\n\n§aClick the coloured box to open the colour wheel.", order = 25)
```
Comment (kristofbolyai) [inline src/main/java/com/wynntils/modules/utilities/configs/UtilitiesConfig.java]: Incorrect text here

Comment (ryanzhoudev) [reply src/main/java/com/wynntils/modules/utilities/configs/UtilitiesConfig.java]: Wait where?

Comment (kristofbolyai) [reply src/main/java/com/wynntils/modules/utilities/configs/UtilitiesConfig.java]: It says searched item highlight color and not guild menu highlight color

Comment (ryanzhoudev) [reply src/main/java/com/wynntils/modules/utilities/configs/UtilitiesConfig.java]: The one for guild territory says the same; I'll change them both

Comment (kristofbolyai) [reply src/main/java/com/wynntils/modules/utilities/configs/UtilitiesConfig.java]: Also perhaps use this color in territory manage overlay aswell if it doesnt have a separate setting

File: src/main/java/com/wynntils/modules/utilities/configs/UtilitiesConfig.java [hunk 2]
```diff
@@ -86,6 +86,12 @@
     @Setting(displayName = "Searched Item Highlight Colour", description = "What colour should the highlight for searched items be?\n\n§aClick the coloured box to open the colour wheel.", order = 23)
     public CustomColor guildTerritoryMenuSearchHighlightColor = new CustomColor(80, 242, 242);
 
+    @Setting(displayName = "Show Guild Manage Member Search Bar", description = "Should the search bar be shown in the guild manage members GUI?\n\n§8Members that match the search will be highlighted.", order = 24)
```
Comment (CringeAura) [inline src/main/java/com/wynntils/modules/utilities/configs/UtilitiesConfig.java]: Do you know if the name of the setting is getting truncated in the menu? If it is, it should be shortened.

Comment (ryanzhoudev) [reply src/main/java/com/wynntils/modules/utilities/configs/UtilitiesConfig.java]: Yup a few of these are getting cut - fixed in next commit

**Review by kristofbolyai — APPROVED** [review]

File: src/main/java/com/wynntils/modules/utilities/overlays/inventories/GuildMemberManageOverlay.java [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+            return;
+        }
+
+        // Process the search field click interaction
+        searchField.mouseClicked(offsetMouseX, offsetMouseY, e.getMouseButton());
+
+        // Only listen for left clicks
+        if (e.getMouseButton() != 0) {
+            return;
+        }
```
Comment (HighCrit) [inline src/main/java/com/wynntils/modules/utilities/overlays/inventories/GuildMemberManageOverlay.java]: Should this be higher in the function?

**Review by HighCrit — CHANGES_REQUESTED** [review]

**Review by HighCrit — APPROVED** [review]

**Pull request merged**